internal/captain/command.go

```diff
@@ -502,11 +502,16 @@ func (c *Command) runner(cobraCmd *cobra.Command, args []string) error {
 	defer profile.Measure(fmt.Sprintf("captain:runner"), time.Now())
 
 	subCommandString := c.UseFull()
-	logging.CurrentCmd = appEventPrefix+subCommandString
+	logging.CurrentCmd = appEventPrefix + subCommandString
 
-	// Send  GA events unless they are handled in the runners...
+	// Send GA events unless they are handled in the runners...
 	if c.analytics != nil {
-		c.analytics.Event(anaConsts.CatRunCmd, appEventPrefix+subCommandString)
+		var label string
+		if len(args) > 0 && (args[0] == constants.PpmShim || args[0] == constants.PipShim) {
```

Comment:
This doesn't seem right, it would run as `state run ppm` I believe, making it `args[2]`. Could you manually test and ensure this is correct?

Reply:
Yep, I manually tested this before opening the PR. I think the first argument is being consumed to determine what the command is. Added an integration test so we can verify.